### ISSUE TITLE
[Input] Missing autocomplete attribute

### ISSUE THREAD

**Issue by Juliecottu** [LuccaSA/lucca-front#3117]
We need the ability to add and populate the HTML `autocomplete` attribute on inputs. 
Additionally, this will prevent Google Chrome from autofilling wrong data
